src/wp-includes/media.php

```diff
@@ -5455,6 +5455,20 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	/*
+	 * Skip programmatically created images within post content as they need to be handled together with the other
+	 * images within the post content.
+	 * Without this clause, they would already be counted below which skews the number and can result in the first
+	 * post content image being lazy-loaded only because there are images elsewhere in the post content.
+	 */
+	static $special_content_contexts = array(
```

Comment:
Why is this static?

Reply:
Because it never changes, so recreating the array every time this function is called is unnecessary. Just a tiny efficiency improvement.

Reply:
Static variables take up memory right? Is this worse than recreating this tiny variable on every run.

Reply:
Fair point, though it's a tradeoff of memory vs time. Using `static` is faster, not using `static` saves memory.

In any case, the impact on both is so tiny that it probably doesn't matter. Happy to remove the `static` here.

Reply:
Realistically, these type of constant values should be declared as a class constant, but that's kind of difficult for a global function...

Reply:
@jrfnl Do you have any thoughts / preferences on using `static` here or not as a workaround? Based on @spacedmonkey's feedback I'm inclined to remove it, since the impact is probably tiny either way. But curious to hear if you have a preference in this odd scenario.

Reply:
The `static` may make testing harder if it's an array which changes (which I understood it's not).
The `static` is also not really used for it's intended purpose.

All in all, with an array with only two entries, I don't think the `static` has added value (or a positive performance impact), so I would remove it.

Even more so, as the array is only used once and only contains two entries, I personally wouldn't even declare it, but would do a straight comparison, i.e. `if ( $a === 'value1' || $a === 'value2' ) {}`.

Reply:
The straight comparison is what this code had originally, but then we got into the weeds of using some sort of array for it. Makes sense that for 2 items like this a simple comparison is probably most straightforward.

Reply:
It is.

Reply:
Reverted in https://github.com/WordPress/wordpress-develop/pull/4300/commits/a2096b96de15aaafa5c48b26312883e5e1d5f365